src/components/atoms/responsive-grid/styles.css

```diff
@@ -0,0 +1,5 @@
+.react-grid-item.react-grid-placeholder {
+  /*background: #efefef;*/
+  /*border: 2px dashed grey;*/
+  border-radius: 4px;
+}
```

Comment:
You can play with the grid styling here. Maybe change the bg color or leave as is

Reply:
we have CardRoot styled component, do we need this styling here?